ios/iosApp/AppDelegate.swift

```diff
@@ -4,47 +4,49 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-    var window: UIWindow?
+	var window: UIWindow?
```

Comment:
It’s actually now _tabs_ (instead of spaces).

> - XCode -> Preferences... ~> Text Editing ~> Indentation => Select 'Prefer Indent Using': Tabs

Reply:
According to [Apple’s own documentation](https://github.com/apple/swift/blob/main/docs/SwiftIndent.md), it looks like spaces should be used by default (because `-use-tabs` is an option).

Reply:
Sur GitHub les tabs prennent plus d'espace mais sur Xcode ça prend pas beaucoup de place. Perso j'aime mieux les tabs, est-ce que je change pour les espaces? (Dsl du délai, complètement oublié).

Reply:
We should definitely use spaces 🙏

Reply:
It now uses 4 spaces for indent. Should be good to merge if there weren't other issues!

Reply:
Lets do this! 🚀